persistence-modules/spring-data-dynamodb/pom.xml

```diff
@@ -174,6 +174,7 @@
         <dynamodblocal.version>1.21.1</dynamodblocal.version>
         <maven-dependency-plugin.version>3.1.1</maven-dependency-plugin.version>
         <spring-boot.version>2.4.7</spring-boot.version>
+        <mockito.version>4.4.0</mockito.version>

```

Comment:
Does 5.11.0 give failure?

Reply:
Yes, this is a quick fix to run the build. Once the build passes we can investigate the code changes required with the new version.

Reply:
OK, I will create a separate task for this module.

Reply:
https://team.baeldung.com/browse/JAVA-33817